src/app/data-model-provider/ProviderMetadataTree.h

```diff
@@ -51,6 +52,9 @@ class ProviderMetadataTree
     virtual CHIP_ERROR EndpointUniqueID(EndpointId endpointId, MutableCharSpan & EndpointUniqueId) = 0;
 #endif
 
+    // Fetch event metadata for a specific event
```

Comment:
This still needs API documentation so that callers and implementors do not have different expectations.